specs/v1.yaml

```diff
@@ -422,6 +422,441 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Webhook"
+        '400':
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          $ref: "#/components/responses/AuthenticationError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '404':
+          $ref: "#/components/responses/NotFoundError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+    delete:
+      summary: Delete Webhook
+      description: Delete a Webhook
+      operationId: delete-webhook
+      security:
+        - auth0:
+          - "delete:webhooks"
+      parameters:
+        - name: id
+          in: path
+          description: the unique id of the nonprofit
+          schema:
+            type: string
+          required: true
+      responses:
+        '200':
+          description: "OK"
+          headers:
+            X-Request-Id:
+              $ref: "#/components/headers/X-Request-Id"
+        '400':
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          $ref: "#/components/responses/AuthenticationError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '404':
+          $ref: "#/components/responses/NotFoundError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+  /v1/webhooks/{id}/test:
+    post:
+      summary: Test Webhook
+      description: Execute a test on a webhook. This will send a test payload to the webhook endpoint
+      operationId: test-webhook
+      security:
+        - auth0:
+          - "read:webhooks"
+      parameters:
+        - name: id
+          in: path
+          description: the unique id of the webhook
+          schema:
+            type: string
+          required: true
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                eventType:
+                  type: string
+                  description: |-
+                    The event type to trigger
+      responses:
+        '200':
+          description: "OK"
+          headers:
+            X-Request-Id:
+              $ref: "#/components/headers/X-Request-Id"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Webhook"
+        '400':
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          $ref: "#/components/responses/AuthenticationError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '404':
+          $ref: "#/components/responses/NotFoundError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+  /v1/webhooks/{id}/messages:
+    get:
+      summary: List Webhook Messages
+      description: Lists the Webhook messages for a specific endpoint.
```

Comment:
the use of endpoint here is not consistent with our RESTful APIs. I would replace endpoint with `webhook`

Reply:
done